.github/workflows/goreleaser.yml

```
@@ -53,9 +53,32 @@ jobs:
         with:
           cosign-release: v2.2.4
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
+
       - name: Install Syft
         uses: anchore/sbom-action/download-syft@b6a39da80722a2cb0ef5d197531764a89b5d48c3 # v0.15.8
 
+      - name: Login to Docker Hub
+        if: startsWith(github.ref, 'refs/tags/')
+        uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20 # v3.1.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Pull base container image
+        run: docker pull cgr.dev/chainguard/static:latest
+
+      - name: Verify base container image
+        run: |
+          cosign verify \
+          --certificate-oidc-issuer=https://token.actions.githubusercontent.com \
+          --certificate-identity=https://github.com/chainguard-images/images/.github/workflows/release.yaml@refs/heads/main \
+          cgr.dev/chainguard/static:latest
```

There shouldn't be a need for the fold operator or trailing slashes. A string extending over multiple lines should automatically be joined with a single space on one line

```suggestion
        run: cosign verify 
          --certificate-oidc-issuer=https://token.actions.githubusercontent.com 
          --certificate-identity=https://github.com/chainguard-images/images/.github/workflows/release.yaml@refs/heads/main
          cgr.dev/chainguard/static:latest
```